functions/createZombie.js

```diff
@@ -13,13 +11,11 @@ module.exports.handler = async (event, context, callback) => {
   log.info('Context => ', context);
 
   const {
-    body: {
-      name,
-    },
+    body: { name },
```

Comment:
troche dziwna konstrukcja tego obiektu - popraw mnie jesli sie myle:

const { body { name } } = event ? jesli mamy tylko jedno property name, to chyba niepotrzebne robienie z tego obiektu ;)